Tests/MongoSwiftTests/MongoCollectionTests.swift

```diff
@@ -173,8 +173,9 @@ final class MongoCollectionTests: XCTestCase {
 
     func testDistinct() throws {
         let distinct = try coll.distinct(fieldName: "cat", filter: [:])
-        expect((distinct.next()?["values"] as? [String])!.sorted()).to(equal(["cat", "dog"]))
-        expect(distinct.next()).to(beNil())
+        expect((distinct as? [String])?.sorted()).to(equal(["cat", "dog"]))
+        // nonexistent field
+        expect(try self.coll.distinct(fieldName: "abc", filter: [:])).to(beEmpty())
```

Comment:
Would it make sense to demonstrate that `null` may be returned as one of several distinct values? IIRC, that was one of the reasons for opening [SWIFT-149](https://jira.mongodb.org/browse/SWIFT-149) in the first place.